src/app/utils/index.ts

```diff
@@ -63,6 +63,12 @@ export function getAlbyAccountName(info: GetAccountInformationResponse) {
   return info.email || info.lightning_address || "getalby.com";
 }
 
+export function isBitcoinAddress(address: string) {
+  return /^(?:[13]{1}[a-km-zA-HJ-NP-Z1-9]{25,34}|(bc1|tb1)[a-z0-9]{39,59})$/i.test(
```

Comment:
where does this come from? should we add a comment here?

Reply:
:+1: https://stackoverflow.com/questions/21683680/regex-to-match-bitcoin-addresses + slightly modified to support testnet addresses